.gitignore

```diff
@@ -1 +1,3 @@
 CMakeLists.txt.user*
+*.pro
```

Comment:
Can we remove these changes, I am not sure we need the first line even, developers can add development environment specific excludes to their global configuration for git, rather than every project.